### PR TITLE
Issue/185 calculate grades

### PR DESCRIPTION
**Pull request by ljoks**
currently uses mock users. that will have to change when we figure out how to get users from an LTI. 

calculates grades for 25 mock users, `userId`s 1-25

### CODE REVIEW

File: backend/src/@types/models/Question.d.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ interface QuestionAttributes {
 	participationPoints: number;
 	correctnessPoints: number;
 	type: string;
+	QuestionOptions: Array<QuestionOption>;
```
Comment (kennygperez) [inline backend/src/@types/models/Question.d.ts:12]: Change this to `QuestionOption[]`

File: backend/src/@types/models/SessionGrade.d.ts [hunk 2]
```diff
@@ -2,15 +2,19 @@ type SessionGradeId = number;
 
 interface SessionGradeAttributes {
 	id: SessionGradeId;
+	courseId: CourseId;
 	userId: UserId;
 	sessionId: SessionId;
+	QuestionGrades: Array<QuestionGrade>;
```
Comment (kennygperez) [inline backend/src/@types/models/SessionGrade.d.ts:8]: Same here

File: backend/src/@types/models/SessionGrade.d.ts [hunk 3]
```diff
@@ -2,15 +2,19 @@ type SessionGradeId = number;
...
 	points: number;
 	maxPoints: number;
 }
 
 interface SessionGradeCreationAttributes {
+	courseId: CourseId;
 	userId: UserId;
 	sessionId: SessionId;
+	QuestionGrades: Array<QuestionGrade>;
```
Comment (kennygperez) [inline backend/src/@types/models/SessionGrade.d.ts:17]: And here

File: backend/src/handlers/question.ts [hunk 4]
```diff
@@ -36,6 +36,7 @@ const get = async (event: APIGatewayEvent): Promise<ProxyResult> => {
 // POST /api/v1/question
 const create = async (event: APIGatewayEvent): Promise<ProxyResult> => {
 	const body = JSON.parse(event.body || '{}');
+	console.log(body);
```
Comment (kennygperez) [inline backend/src/handlers/question.ts:39]: Remove console log

File: backend/src/handlers/sessionGrades.ts [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+	QuestionUserResponse,
+	SessionGrade,
+	QuestionGrade,
+	Question,
+	Session,
+	QuestionOption,
+} from '../models';
+import responses from '../util/api/responses';
+
+// import fetch from 'node-fetch';
```
Comment (kennygperez) [inline backend/src/handlers/sessionGrades.ts:12]: Remove commented code

**Review by kennygperez — CHANGES_REQUESTED** [review]
Just change a few things, super ez

**Review by Goff-Davis — COMMENTED** [review]
Looks good other than the changes kenny requested 👍

**Review by Goff-Davis — DISMISSED** [review]

**Pull request merged**